desktop-widgets/filterwidget2.cpp

```diff
@@ -52,21 +55,16 @@ FilterWidget2::FilterWidget2(QWidget* parent) :
 	connect(ui.minVisibility, &StarWidget::valueChanged,
 		this, &FilterWidget2::updateFilter);
 
-	// We need these insane casts because Qt decided to function-overload some of their signals(!).
```

Comment:
Please keep at least the part of the comment that explains why `QOverload` is needed. Most of us are *not* Qt experts!

Reply:
ok, but that could be used in more places afterwards so the new style commit style can be used.